### PR TITLE
Different heading for continue as user in signup

### LINKED ISSUE #86806: Continue as user: Heading is misleading
### Details

Context: pdt2If-15A-p2

https://wordpress.com/start/hosting/user-social?ref=developer-lp&flow=new-hosted-site

image

[Logged in only ] Let's get you signed up. heading is miss-leading. I am already signed up and logged in.

### Checklist

_No response_

### Related

_No response_

### PR DESCRIPTION
**Pull request by escapemanuele**
Context: pdt2If-15A-p2#comment-719

## What

If you try to signup while logged in, you see

image

Buy you are already signed up, and this could be confusing.

## Solution

image

## Testing 

1. Visit `[Live Link]/start/hosting/user-social?ref=developer-lp&flow=new-hosted-site`
2. Be sure to be logged in and check the heading
3. Check that the translation check works

Fixes https://github.com/Automattic/wp-calypso/issues/86806

### CODE REVIEW

File: client/signup/steps/user/index.jsx
Comment (Aurorum) [inline client/signup/steps/user/index.jsx:473]: Purely out of interest for myself, would you mind explaining the logic here around only showing this for English locales? Is this maybe missing a `hasTranslation` or am I misunderstanding it? :)

File: client/signup/steps/user/index.jsx
Comment (Aurorum) [inline client/signup/steps/user/index.jsx]: Not sure if the copy is final, but I'd suggest that "Is this you?" would make more sense given the UI here.

File: client/signup/steps/user/index.jsx
Comment (escapemanuele) [reply client/signup/steps/user/index.jsx:473]: You're totally right! I tested `hasTranslation` a little and couldn't get it to work properly, it was returning false for already existing translations 🤔  So I went on and thought about simply doing like this and then removing the `englishLocales` check once translations were ready!

File: client/signup/steps/user/index.jsx
Comment (escapemanuele) [reply client/signup/steps/user/index.jsx]: Yes, it makes total sense, I just updated the copy 👍

File: client/signup/steps/user/index.jsx
Comment (Aurorum) [reply client/signup/steps/user/index.jsx:473]: Ah, got it! Rather than re-visiting this (and potentially forgetting...), one other option could be to just add a String Freeze and wait for the translations here.

Although, as I think about this more, I wonder if it might make sense to use the strings from `/log-in` instead? Consistency and no new translations needed 😁 Obviously your call though!

Screenshot 2024-01-25 at 19 43 10

Comment (Aurorum) [inline client/signup/steps/user/index.jsx:473]: Nit (obvs!): maybe just me, but two if statements feels a bit weird for the same return.

**Review by Aurorum — APPROVED** [review]
Not sure if you actually want my review here since a non-a11n approval might not be sufficient for merging! But this tests well for me - left two comments. 

Screenshot 2024-01-25 at 19 47 59

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/11032647

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @escapemanuele for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/11032647).

**Pull request merged**
